packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -320,6 +320,31 @@ const PRESETS = [
         assets: ['translations'],
         private: true
     },
+    {
+        id: 'retail-react-app-bug-bounty',
+        name: 'Retail React App Bug Bounty Project',
+        description: '',
+        templateSource: {
+            type: TEMPLATE_SOURCE_NPM,
+            id: '@salesforce/retail-react-app'
+        },
+        questions: [...EXTENSIBILITY_QUESTIONS, ...HYBRID_QUESTIONS, ...RETAIL_REACT_APP_QUESTIONS],
+        answers: {
+            ['project.extend']: true,
+            ['project.hybrid']: true,
```

Comment:
Should this be false? The scope is to test Retail React App, not Hybrid scenarios.


```suggestion
            ['project.hybrid']: false,
```

Reply:
Good point. This is a PWA only site so I'll change this.